pkg/oc/bootstrap/docker/host/host.go

```diff
@@ -78,7 +102,10 @@ func (h *HostHelper) CopyToHost(src, dst string) error {
 	return nil
 }
 
-func (h *HostHelper) CopyFromHost(src, dst string) error {
+// CopyFromRemoteHost copies the remote host directory into local filesystem.
+// This is used when Docker runs in VM.
+// TODO: Do we actually need this? We are not syncing anything back from the remote host.
```

Comment:
Give it time.  Ugly will spread and you'll be trying to figure out whether you're in sync or not, which "extra" files you have from previous runs, whether you should merge those files, what needs to be deleted when new directories are made, etc.

Reply:
i have no doubt the ugly will spread and somebody will want to sync the data they generated by their components remotely back to local baseDir...